.pre-commit-config.yaml

```diff
@@ -33,6 +33,15 @@ repos:
       types:
         - python
 
+    - id: ruff-isort
+      name: ruff-isort
+      entry: ruff check --select I001 --fix
+      language: system
+      pass_filenames: true
+      files: "./"
+      types:
+        - python
```

Comment:
Given that this PR enables `I001` in `pyproject.toml`, is adding this required? I think the `ruff` hook defined below should do the work. If we want to make the `isort` errors "fixable", we can add `fixable = ["I001"]` to `pyproject.toml` to do the work for us.

Reply:
oh I didn't know, thanks for point me to that. I will apply that, thank you so much.

Reply:
oh just saw that you already suggested that change .. just accepted it and push your changes. thanks